### DISCUSSION TITLE
Why using Grid.Col error on next js app router ?

### DISCUSSION THREAD

**Discussion by zeffrypfs** [mantinedev/mantine#7125] [Q&A] [answered]
Like the title above, when using Grid.Col in Next JS App Router it's throw error : 
```
Error: Cannot access Grid.Col on the server. You cannot dot into a client module from a server component. You can only pass the imported name through.
```

I'm know to fix this i should write "use client" at the top of file. But i'm curious why ?

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/guides/next/#compound-components-in-server-components

**Reply by zeffrypfs** [reply]
Thank you for the link 🙌
